packages/http-client-js/src/components/operation-handlers/paginated-operation-handler.tsx

```diff
@@ -1,29 +1,80 @@
 import * as ay from "@alloy-js/core";
 import { Children } from "@alloy-js/core";
 import { $ } from "@typespec/compiler/experimental/typekit";
-import * as ts from "@typespec/emitter-framework/typescript";
+import * as ts from "@alloy-js/typescript";
 import { HttpOperation } from "@typespec/http";
 import { OperationHandler } from "./types.js";
 import { ClientOperation } from "../client-operation.jsx";
+import { HttpRequest } from "../http-request.jsx";
+import { HttpResponse } from "../http-response.jsx";
+import { OperationOptionsDeclaration } from "../operation-options.jsx";
+import { getPageSettingsTypeRefkey, PageSettingsDeclaration } from "./paging/page-settings.jsx";
+import * as cl from "@typespec/http-client";
+import { reportDiagnostic } from "../../lib.js";
+import { getClientcontextDeclarationRef } from "../client-context/client-context-declaration.jsx";
+import { getOperationParameters } from "../operation-parameters.jsx";
+import { FunctionDeclaration, TypeExpression } from "@typespec/emitter-framework/typescript";
+import { PageResponseDeclaration } from "./paging/page-response.jsx";
 
 export const PaginatedOperationHandler: OperationHandler = {
   canHandle(httpOperation: HttpOperation): boolean {
     const pagingOperation = $.operation.getPagingOperation(httpOperation.operation);
     return pagingOperation !== undefined;
   },
   handle(httpOperation: HttpOperation): Children {
+    const clientLibrary = cl.useClientLibrary();
+    const client = clientLibrary.getClientForOperation(httpOperation);
+
+    if (!client) {
+      reportDiagnostic($.program, {
+        code: "client-not-found",
+        target: httpOperation.operation,
+      });
+      return;
+    }
     const internalOperationRefkey = ay.refkey(httpOperation.operation, "internal");
+    const pagingOperation = $.operation.getPagingOperation(httpOperation.operation)!;
+
+    // build response keys
+    const responseRefkey = ay.refkey(httpOperation, "http-response");
+    const _pageSettings = pagingOperation?.input;
+    console.log(_pageSettings);
     const operationRefkey = ay.refkey(httpOperation.operation);
+    // Question here: how to get the refkey for the response type? 
+    // const responseType = getPageResponseTypeRefkey(httpOperation).key;
```

Comment:
I want to build the paging return type which is something string like `<PagedAsyncIterableIterator<Foo, LinkPageResponse, LinkPageSettings>>`, then how do I get the refKey for response definition? How can I use it as a string?